apps/ledger-live-desktop/tests/page/drawer/ledger.sync.drawer.ts

```diff
@@ -31,7 +31,7 @@ export class LedgerSyncDrawer extends Drawer {
   @step("Open the Manage Key section")
   async manageBackup() {
     await expect(this.manageBackupButton).toBeVisible();
-    await this.manageBackupButton.click();
+    await this.manageBackupButton.click({ force: true });
```

Comment:
Do it change something to force ?

Reply:
I add the force due to the error we had in local when Playwright didn't click on `manageBackupButton`